index.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+module.exports = {
+    extends: 'stylelint-config-standard',
+    plugins: [
+        'stylelint-order',
+    ],
+    rules: {
+        indentation: 4,
+        'selector-pseudo-element-colon-notation': 'single',
+        'max-nesting-depth': [2, {
+            ignore: ['blockless-at-rules'],
+        }],
+        'no-duplicate-selectors': true,
+        'no-unknown-animations': true,
+        'number-max-precision': 2,
+        'unit-blacklist': ['rem'],
+        'value-no-vendor-prefix': true,
+        'property-no-vendor-prefix': true,
+        'selector-max-compound-selectors': 3,
+        'selector-max-id': 1,
+        'selector-max-specificity': '1,2,0',
+        'selector-max-type': 0,
+        'selector-max-universal': 0,
+        'selector-no-qualifying-type': true,
+        'selector-no-vendor-prefix': true,
+        'media-feature-name-no-vendor-prefix': true,
+        'at-rule-no-vendor-prefix': true,
+        'at-rule-no-unknown': [true, {
+            ignoreAtRules: ['mixin', 'include', 'each'],
+        }],
+        'font-weight-notation': 'named-where-possible',
+        'function-url-quotes': 'never',
+        'string-quotes': 'double',
+        'value-keyword-case': 'lower',
+        'block-opening-brace-newline-after': 'always',
+        'block-closing-brace-newline-before': 'always',
+        'selector-attribute-quotes': 'always',
+        'selector-list-comma-newline-before': 'never-multi-line',
+        'at-rule-semicolon-newline-after': 'always',
+        'max-line-length': [120, {
+            ignore: ['comments'],
+        }],
+        'order/order': [
+            'dollar-variables',
+            'at-rules',
+            'declarations',
+            'rules',
+        ],
+        'order/properties-order': [
+            // Positioning
+            'position',
+            'top',
+            'right',
+            'bottom',
+            'left',
+            'z-index',
+
+            // Display & Box-model
```

Comment:
I tried to demand a blank line between each group, unfortunate the linter does not support this feature. 😢